### ISSUE TITLE
I got a error in Que 10 Python 3 Solution.

### ISSUE THREAD

**Issue by Utkarsh4697** [darkprinx/100-plus-Python-programming-exercises-extended#45]
When we use more than 2 duplicates than the program gives us an incorrect solution. the program only works on 2 duplicates. so I think we need to use a While loop instead of if statement.

**Comment by darkprinx** [comment]
Yes, I just noticed this factor.  You just got it right. Can you please fix those codes out there and give a pull request. It will be a very kind help :smiley:

**Comment by darkprinx** [comment]
Confirmed and fixed in e0c4c21 — both Python 3 solutions iterated over the list while removing from it, which only works reliably for exactly 2 duplicates (skips elements on 3+ due to the list shifting during iteration). Fixed by iterating over set(word) and looping until each duplicate is fully removed. Verified against the sample plus 3- and 5-duplicate test cases. Thanks for the report!
